<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndMain</class>
 <widget class="QMainWindow" name="WndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>As</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QFrame" name="frmMain">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>45</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpenProject"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionNew"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionPreferences"/>
   <addaction name="separator"/>
   <addaction name="actionConsole"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStepInto"/>
   <addaction name="actionStepOver"/>
   <addaction name="actionStepOut"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionOpenFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/res/Open.png</normaloff>:/images/res/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/ico_add_prj</normaloff>:/images/ico_add_prj</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/ico_save_all</normaloff>:/images/ico_save_all</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/ico_save_as</normaloff>:/images/ico_save_as</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/ico_exit</normaloff>:/images/ico_exit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/res/setting.png</normaloff>:/images/res/setting.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Change the source code editor configuration</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/res/konsole.png</normaloff>:/images/res/konsole.png</iconset>
   </property>
   <property name="text">
    <string>View console</string>
   </property>
   <property name="toolTip">
    <string>Shows or hides the console frame</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionActionProjectExplorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Project explorer</string>
   </property>
   <property name="toolTip">
    <string>Show or hides the project explorer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/images/ico_info</normaloff>:/images/ico_info</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset>
     <normaloff>:/images/ico_open_prj</normaloff>:/images/ico_open_prj</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/run_and_debug/res/run.png</normaloff>:/run_and_debug/res/run.png</iconset>
   </property>
   <property name="text">
    <string>actionRun</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStepInto">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/run_and_debug/res/debug-step-into.png</normaloff>:/run_and_debug/res/debug-step-into.png</iconset>
   </property>
   <property name="text">
    <string>actionStep</string>
   </property>
   <property name="toolTip">
    <string>Step</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStepOver">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/run_and_debug/res/debug-step-over.png</normaloff>:/run_and_debug/res/debug-step-over.png</iconset>
   </property>
   <property name="text">
    <string>actionStepOver</string>
   </property>
   <property name="toolTip">
    <string>Step Over</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStepOut">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/run_and_debug/res/debug-step-out.png</normaloff>:/run_and_debug/res/debug-step-out.png</iconset>
   </property>
   <property name="text">
    <string>actionStepOut</string>
   </property>
   <property name="toolTip">
    <string>Step Out</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="luanda.qrc">
     <normaloff>:/run_and_debug/res/stop.png</normaloff>:/run_and_debug/res/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Execution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="luanda.qrc"/>
 </resources>
 <connections/>
</ui>
